src/adapter.js

```diff
@@ -192,10 +317,20 @@ export function parse({minim, source}, done) {
         }
 
         // Currently, default responses are not supported in API Description format
-        const relevantResponses = _.omit(methodValue.responses, 'default');
+        // TODO: [Annotation] Add warning about not showing the default!
+        const relevantResponses = _.chain(methodValue.responses)
+          .omit('default')
+          .omit(isExtension)
+          .value();
 
         if (_.keys(relevantResponses).length === 0) {
-          createTransaction(minim, transition, method);
+          if (bodyParameters.length) {
+            // Create an empty successful response so that the request/response
+            // pair gets properly generated.
+            relevantResponses[200] = {};
```

Comment:
My concern here is that we're making some assumption from the document, when I believe other tooling should know what to do when there is no response found.